src/main/java/frc/robot/commands/ShooterPID.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterSystem;
+
+public class ShooterPID extends Command {
+    private ShooterSystem shooterSystem;
```

Comment:
```suggestion
    private final ShooterSystem shooterSystem;
```

we should define the class members as final if we can.

Reply:
done